airflow/utils/task_group.py

```diff
@@ -543,6 +554,13 @@ def iter_tasks(self) -> Iterator[AbstractOperator]:
                         f"Encountered a DAGNode that is not a TaskGroup or an AbstractOperator: {type(child)}"
                     )
 
+    def add_to_taskgroup(self, task_group: TaskGroup) -> None:
+        """No-op, since we're not a task.
+
+        :meta private:
+        """
+        ...
```

Comment:
Probably needs some more text to explain why not being a task makes this a no-op. It’s not immediately obvious why an edge modifier should not be added to a task group.

Also the `...` is not needed.